cmd/check_container.go

```diff
@@ -163,7 +162,7 @@ var checkContainerCmd = &cobra.Command{
 			// get the project info from pyxis
 			certProject, err := pyxisClient.GetProject(ctx)
 			if err != nil {
-				log.Error(fmt.Errorf("%w: %s", errors.ErrRetrievingProject, err))
+				log.Error(fmt.Errorf("%w: %s", ErrRetrievingProject, err))
```

Comment:
Is the wrap ordering incorrect here?

Reply:
Probably not. I hadn't gotten to the "fix all the wrapping" phase. Otherwise, this PR would likely touch literally every file. I'm not looking to solve all the things in this particular PR.